cmd/build/helmify/main.go

```diff
@@ -133,6 +133,10 @@ func (ks *kindSet) Write() error {
 				obj = strings.Replace(obj, "      labels:", "      labels:\n{{- include \"gatekeeper.podLabels\" . }}", 1)
 			}
 
+			if name == "gatekeeper-manager-role" && kind == "Role" {
+				obj += "{{- with .Values.controllerManager.extraRules }}\n  {{- toYaml . | nindent 0 }}\n{{- end }}\n"
```

Comment:
nit: can you follow similar format as
https://github.com/open-policy-agent/gatekeeper/blob/b5ab5608421d415f34008e399a02b4a57a9e511f/cmd/build/helmify/replacements.go#L154-L157 to be consistent?

Reply:
Thank you for your comment, @ritazh. The approach you are proposing seems to be valid only for Kuberentes resources that are defined in the `kustomize-for-helm.yaml` file. Unfortunately this is not the case for the `Role` called `gatekeeper-manager-role`. But I might be wrong and I just don't understand what you are asking for. If that's the case, please provide more details so I can amend the PR.

Reply:
To clarify, I was thinking, to be consistent with other templates, you could do something like this? the spacing might need a bit more testing. WDYT? 
```go
obj += "  {{- range .Values.controllerManager.extraRules }}\n  - {{ . }}\n  {{- end }}\n"
```

Reply:
I think that using the `range` for walking through the list of maps where each of the map has to be turned to YAML anyway makes it more complicated than just to turn the whole list into YAML directly.

Reply:
that makes sense.